packages/valory/skills/abstract_round_abci/abci_app_chain.py

```diff
@@ -64,7 +64,7 @@ def chain(  # pylint: disable=too-many-locals
         "Found multiple occurences of same Abci App",
     )
     non_abstract_abci_apps = [
-        abci_app.__name__ for abci_app in abci_apps if not abci_app.is_abstract
+        abci_app.__name__ for abci_app in abci_apps if not abci_app.is_abstract()
```

Comment:
we're dealing with a class, not an instance, hence the property is not usable @Adamantios